src/main/java/mate/academy/lib/Injector.java

```diff
@@ -47,8 +48,8 @@ private Class<?> findClassExtendingInterface(Class<?> certainInterface) {
             }
         }
         throw new RuntimeException("Can't find class which implements "
-                + certainInterface.getName()
-                + " interface and has valid annotation (Dao or Service)");
+            + certainInterface.getName()
+            + " interface and has valid annotation (Dao or Service)");
```

Comment:
The error message mentions 'valid annotation (Dao or Service)', but the code only checks for the Dao annotation. If there is an intention to support other annotations like Service, the code should be updated to reflect that.